hamid914-shahrekord.md

```diff
@@ -0,0 +1,5 @@
+## Shahrekord, Chaharmahal-O-Bakhtiari
+
+### Places to eat
```

Comment:
@hamid914 super excited to see you in class today! What you recommend as restaurants and foods someone "must try" while they are in town?